spec/presenters/two_factor_auth_code/phone_delivery_presenter_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe TwoFactorAuthCode::PhoneDeliveryPresenter do
+  let(:presenter) do
+    TwoFactorAuthCode::PhoneDeliveryPresenter.new(
+      code_value: '123',
+      delivery_method: 'sms',
+      phone_number: '123-123-1234'
+    )
+  end
+
+  it 'implements GenericDeliveryPresenter' do
+    TwoFactorAuthCode::PhoneDeliveryPresenter.instance_methods(false).each do |m|
+      expect(presenter.send(m)).to be
+    end
+  end
+end
```

Comment:
This test looks like it wasn't completed. I would expect this spec to test this presenter's public methods (header, help_text, fallback_links). If we also want to make sure it inherits from GenericDeliveryPresenter, we can add an expectation such as:

```ruby
expect(presenter.class.superclass).to eq TwoFactorAuthCode::GenericDeliveryPresenter
```